### PR TITLE
Dobrin and Dmitri language review

### PR DESCRIPTION
**Pull request by dmpop**
### PR creator: Description

Language review of the Deployment Guide by Dobrin, with contributions from yours truly.

### PR creator: Are there any relevant issues/feature requests?

* bsc#1188701

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Review by cwickert — COMMENTED** [review]
@dariavladykina, I'm handing this over to you with roughly half of the files reviewed. Please see my comments for details.

File: xml/entity-decl.ent [hunk 1]
```diff
@@ -621,7 +621,7 @@ they also have a trademark policy, let's be careful here. -->
         <para>
          Extends the functionality of &productname; with packages from &sled;,
          like additional desktop applications (office suite, e-mail client,
-         graphical editor, etc.) and libraries. It allows to combine both
+         graphical editor, etc.) and libraries. It lets you to combine both
```
Comment (dariavladykina) [inline xml/entity-decl.ent:624]:
```suggestion
         graphical editor, etc.) and libraries. It lets you combine both
```

**Review by dariavladykina — COMMENTED** [review]
Review updates.

File: xml/deployment_yast_installer.xml [hunk 2]
```diff
@@ -870,22 +866,22 @@
...
-     After &productname; has been successfully registered, you
-     are asked whether to install the latest available online updates during
-     the installation. If choosing <guimenu>Yes</guimenu>, the system will be
-     installed with the most current packages without having to apply the
-     updates after installation. Activating this option is recommended.
+     After &productname; has been successfully registered, you are asked whether
+     to install the latest available online updates during the installation. If
+     you choose <guimenu>Yes</guimenu>, the system will installed the most
+     current packages without having to apply updates after installation. It is
+     recommended to enable this option.
```
Comment (dariavladykina) [inline xml/deployment_yast_installer.xml:873]:
```suggestion
     you choose <guimenu>Yes</guimenu>, the system will be installed
     with the most current packages without having to apply updates after
     installation. It is recommended to enable this option.
```

File: xml/deployment_yast_installer.xml [hunk 3]
```diff
@@ -870,22 +866,22 @@
...
 
    <para>
-    If the system was successfully registered during installation, &yast; will
-    disable repositories from local installation media such as CD/DVD or flash
-    disks when the installation has been completed. This prevents problems if
-    the installation source is no longer available and ensures that you always
-    get the latest updates from the online repositories.
+    If the system is successfully registered during installation, &yast;
+    disables repositories from local installation media such as CD/DVD or flash
+    disks when the installation completes. This prevents problems causes by the
```
Comment (dariavladykina) [inline xml/deployment_yast_installer.xml:882]:
```suggestion
    disks when the installation completes. This prevents problems caused by the
```

File: xml/deployment_yast_installer.xml [hunk 4]
```diff
@@ -1072,10 +1068,10 @@ sle-live-patching 8c541494</screen>
...
   <para>
    In this dialog the installer lists modules and extensions that are available
-   for &productname;. Modules are components which allow you to shape the
-   product according to your needs. They are free of charge. Extensions add
-   functionality to your product. They are offered as subscriptions and require
-   a registration key that is liable for costs.
+   for &productname;. Modules are components that allow you to customize the
+   product according to your needs. They are included in your &productname;
+   subscription. Extensions add functionality to your product. They must be
+   purchased separately.
```
Comment (dariavladykina) [inline xml/deployment_yast_installer.xml:1074]:
```suggestion
   for &productname;. Modules are components that allow you to customize the
   product according to your needs are included in your &productname;
   subscription. Extensions that add functionality to your product must be
   purchased separately.
```

File: xml/libvirt_guest_installation.xml [hunk 5]
```diff
@@ -434,10 +435,8 @@
     <listitem>
      <para>
       It is possible to directly specify the Kernel and Initrd of the
-      installer, for example from a network source.
-      <phrase
-      os="sles;sled">To set up a network source, see
-      <xref linkend="sec-deployment-instserver-http"/>.</phrase>
+      installer, for example from a network source. <phrase
```
Comment (dariavladykina) [inline xml/libvirt_guest_installation.xml:438]:
```suggestion
      installer, for example, from a network source. <phrase
```

File: xml/planning.xml [hunk 6]
```diff
@@ -22,8 +22,7 @@
   </dm:docmanager>
  </info>
  <sect1 xml:id="sec-planning-start">
-  <title>Considerations for deployment of <phrase os="sles">a &sls;</phrase><phrase os="slemicro">&slem;</phrase></title>
-
+  <title>Considerations for deployment of <phrase os="sles">&sls;</phrase><phrase os="slemicro">&slem;</phrase></title>
   <para>
    The implementation of an operating system either in an existing IT
    environment or as a completely new roll out must be carefully prepared. At
```
Comment (dariavladykina) [inline xml/planning.xml:28]:
```suggestion
   environment or as a completely new rollout must be carefully prepared. At
```

File: xml/yast2_lang.xml [hunk 7]
```diff
@@ -46,7 +46,7 @@
   than the primary one.
  </para>
  <para>
-  In addition, the &yast; timezone module allows you to adjust your country
+  The &yast; timezone module allows you to adjust your country
   and timezone settings accordingly. It also lets you synchronize your system
```
Comment (dariavladykina) [inline xml/yast2_lang.xml:50]:
```suggestion
  The &yast; time zone module allows you to adjust your country
  and time zone settings accordingly. It also lets you synchronize your system
```

File: xml/yast2_printer.xml [hunk 8]
```diff
@@ -39,62 +37,46 @@
  <note>
   <title>Starting CUPS</title>
   <para>
-   To print from your system, CUPS must be running. In case it is not
-   running, you are asked to start it. Answer with <guimenu>Yes</guimenu>, or
-   you cannot configure printing. In case CUPS is not started at boot time, you
-   will also be asked to enable this feature. It is recommended to say
-   <guimenu>Yes</guimenu>, otherwise CUPS would need to be started manually
-   after each reboot.
+   To use the printer connected to your machine you must have CUPS installed and
```
Comment (dariavladykina) [inline xml/yast2_printer.xml:40]:
```suggestion
   To use the printer connected to your machine, you must have CUPS installed and
```

File: xml/yast2_printer.xml [hunk 9]
```diff
@@ -174,9 +156,9 @@
    <title>Adding drivers with &yast;</title>
    <para>
     Not all printer drivers available for &productname; are installed by
-    default. If no suitable driver is available in the <guimenu>Find and Assign
-    a Driver</guimenu> dialog when adding a new printer install a driver
-    package containing drivers for your printers:
+    default. When adding a printer no suitable driver is available in the
```
Comment (dariavladykina) [inline xml/yast2_printer.xml:159]:
```suggestion
    default. When adding a printer, if no suitable driver is available in the
```

File: xml/yast2_sw.xml [hunk 10]
```diff
@@ -13,11 +13,12 @@
...
-    Use &yast;'s software management module to search for software components
-    you want to add or remove. &yast; resolves all dependencies for you. To
-    install packages not shipped with the installation media, add
-    software repositories to your setup and let &yast; manage them. Keep your
-    system up-to-date by managing software updates with the update applet.
+    Using &yast;'s software management module, you can search for software
+    packages as well as install and remove them. When installing packages,
+    &yast; automatically resolves all dependencies. To install packages that are
+    not on the installation medium, you can add software repositories and &yast;
+    to manage them. You can also keep your system up-to-date by managing
```
Comment (dariavladykina) [inline xml/yast2_sw.xml:20]:
```suggestion
    to manage them. You can also keep your system up to date by managing
```

File: xml/yast2_sw.xml [hunk 11]
```diff
@@ -26,10 +27,10 @@
   </dm:docmanager>
  </info>
  <para>
-  Change the software collection of your system with the &yast; Software
-  Manager. This &yast; module is available in two flavors: a graphical variant
-  for X Window and a text-based variant to be used on the command line. The
-  graphical flavor is described here&mdash;for details on the text-based
+  The &yast; Software Manager makes it possible to manage software sources on
+  you system. There are two versions of this &yast; module: a graphical version
+  for X Window and a text-based version for use with the command line. The
```
Comment (dariavladykina) [inline xml/yast2_sw.xml:32]:
```suggestion
  your system. There are two versions of this &yast; module: a graphical version
  for X Window and a text-based version to use with the command line. The
```

File: xml/yast2_sw.xml [hunk 12]
```diff
@@ -454,9 +455,9 @@
     </step>
     <step>
      <para>
-      In case &yast; found dependencies on other packages, a list of packages
-      that have additionally been chosen for installation, update, or removal is
-      presented. Click <guimenu>Continue</guimenu> to accept them.
+      If &yast; finds additional dependencies,it shows a list of related
```
Comment (dariavladykina) [inline xml/yast2_sw.xml:458]:
```suggestion
      If &yast; finds additional dependencies, it shows a list of related
```

**Review by dariavladykina — COMMENTED** [review]
Suggestions from review.

**Pull request merged**